src/header.rs

```diff
@@ -107,8 +107,7 @@ pub fn parse_frame_header(data: &[u8]) -> Result<FrameHeader, Mp3Error> {
         _ => return Err(Mp3Error::HeaderError),
     };
 
-    // Protected if the bit is 0
-    let protection = header[1] & 0x01_u8 == 0;
+    let protection = header[1] & 0x01_u8 != 0;
```

Comment:
This is protection, not padding. The old version was correct IIRC. No idea whether padding needs to be flipped tho.